Doc/library/functools.rst

```diff
@@ -347,8 +347,7 @@ The :mod:`functools` module defines the following functions:
 
       def partial(func, /, *args, **keywords):
           def newfunc(*more_args, **more_keywords):
-              keywords_union = {**keywords, **more_keywords}
-              return func(*args, *more_args, **keywords_union)
+              return func(*args, *more_args, **keywords | more_keywords)
```

Comment:
This code looks confusing to me, because `**` is closer to `keywords` than `|`. According to the PEP 8 style it looks like `(**keywords) | more_keywords` rather than `**(keywords | more_keywords)`. I think it will be confusing for many readers.

To avoid confusion, you can add parentheses. Or leave the code as it is.